docs/user/CommandLineExamples.asciidoc

```diff
@@ -7,41 +7,48 @@ See the User Guide for more details on command usage.
 === Basic
 
 ==== Display available commands:
+
```

Comment:
This whole document has a bunch of single lines added to not confuse the `vim` highlighter.